51
0 root (MethodDeclaration)
1 body (BlockStmt)
2 statements
3 statement (ExpressionStmt)
4 expression (MethodCallExpr)
5 name (SimpleName)
6 identifier='i'
7 scope (NameExpr)
8 name (SimpleName)
9 identifier='Log'
10 arguments
11 argument (StringLiteralExpr)
12 value='HU'
13 argument (StringLiteralExpr)
14 value='======onFailure sucesss=='
15 statement (ExpressionStmt)
16 expression (MethodCallExpr)
17 name (SimpleName)
18 identifier='onError'
19 scope (NameExpr)
20 name (SimpleName)
21 identifier='callBack'
22 arguments
23 argument (NameExpr)
24 name (SimpleName)
25 identifier='e'
26 argument (NameExpr)
27 name (SimpleName)
28 identifier='mClient'
29 type (VoidType)
30 name (SimpleName)
31 identifier='onFailure'
32 parameters
33 parameter (Parameter)
34 isVarArgs='false'
35 name (SimpleName)
36 identifier='call'
37 type (ClassOrInterfaceType)
38 name (SimpleName)
39 identifier='Call'
40 parameter (Parameter)
41 isVarArgs='false'
42 name (SimpleName)
43 identifier='e'
44 type (ClassOrInterfaceType)
45 name (SimpleName)
46 identifier='IOException'
47 annotations
48 annotation (MarkerAnnotationExpr)
49 name (Name)
50 identifier='Override'
0 1
1 2
2 3
3 4
4 5
5 6
4 7
7 8
8 9
4 10
10 11
11 12
10 13
13 14
2 15
15 16
16 17
17 18
16 19
19 20
20 21
16 22
22 23
23 24
24 25
22 26
26 27
27 28
0 29
0 30
30 31
0 32
32 33
33 34
33 35
35 36
33 37
37 38
38 39
32 40
40 41
40 42
42 43
40 44
44 45
45 46
0 47
47 48
48 49
49 50

Called when the request could not be executed due to cancellation, a connectivity problem or timeout. Because networks can fail during an exchange, it is possible that the remote server accepted the request before the failure.
